CMake/libusb_config.cmake

```diff
@@ -2,13 +2,12 @@ if (NOT TARGET usb)
     find_library(LIBUSB_LIB usb-1.0)
     find_path(LIBUSB_INC libusb.h HINTS PATH_SUFFIXES libusb-1.0)
     include(FindPackageHandleStandardArgs)
-    find_package_handle_standard_args(usb "libusb not found; using internal version" LIBUSB_LIB LIBUSB_INC)
-    if (USB_FOUND AND NOT USE_EXTERNAL_USB)
-        add_library(usb INTERFACE)
+    find_package_handle_standard_args(usb LIBUSB_LIB LIBUSB_INC)
+ 
+    add_library(usb INTERFACE)
+    if (USB_FOUND)
```

Comment:
Who turn this flag on?
When will it be off?

Reply:
I not found place where we create or change this flag, only 1 usage.
Is this unnecessary flag?

Reply:
Read online..

I think `find_package_handle_standard_args(usb` set it

Reply:
Please make sure what happened when it is not found.
Maybe we need an error?

Reply:
> Read online..
> 
> I think `find_package_handle_standard_args(usb` set it

Oh, I Already read about it. USB_FOUND will true if  LIBUSB_LIB LIBUSB_INC will be found.

USB_FOUND is false if `libusb-1-dev` not installed and installation crashing on `install(TARGETS usb EXPORT realsense2Targets)` line

We can add:
`message(FATAL_ERROR "decription")` - stop processing and generation.
or
`message(SEND_ERROR "decription")` - continue processing, but skip generation.

Reply:
So are you saying windows has libusb installed?
And on windows USB_FOUND = true?

Reply:
I see in a logs _LIBUSB_LIB (missing: LIBUSB_INC)_ and the code in _if_ not executing.
But the viewer run after building.
Are you think it may cause error in some places?

Reply:
So it does not get into the if and the install does not complain??

Reply:
Yes, it does not get into the `if`. But the viewer run correctly.